File: uri-judge/[1589]Bob_Conduite.cpp
//Author: Hrz - 2ºSM de CC
//Site: hrezend.github.io

#include <bits/stdc++.h>
#define desync ios_base::sync_with_stdio(false);cin.tie(NULL);cout.tie(NULL)
#define endl '\n'
using namespace std;

int main(){

	desync;
	
	int R1, R2;
	int Casos;
	cin >> Casos;

	for(int i = 0; i < Casos; i++){
		cin >> R1 >> R2;
		
		cout << R1+R2 << endl;
	}
}

File: uri-judge/[1589]Bob_Conduite.java
//Author: Hrz - 2ºSM de CC
//Site: hrezend.github.io

import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner hr = new Scanner(System.in);
        int R1, R2, Casos;
        
        Casos = hr.nextInt();
        
        for(int i = 0; i < Casos; i++){
            R1 = hr.nextInt();
            R2 = hr.nextInt();
            
            System.out.println(R1+R2);
        }
 
    }
 
}
